pgml-dashboard/templates/sql.html

```diff
@@ -24,6 +24,15 @@
       <% } %>
     </tbody>
   </table>
+  <code>
```

Comment:
It would be nice to hide this block if there is only a single statement, so that we don't double display the timing in that case.

Reply:
Added a utils.rs module with a function for doing this. I made it take an `f64`, as we convert it to that regardless of whether we are working with a `std::time::Duration` or `postgres Interval`.

Reply:
Done!